### ISSUE TITLE
Train on FF++  C23 

### ISSUE THREAD

**Issue by wuwuwuyue** [10Ring/LAA-Net#4]
We tried to train on FF++C23, but the training results were very poor. The AUC on CDFv2 was less than 70%. Here is the config file attached. It may be a problem with our processing. Has anyone successfully reproduced the results?
![屏幕截图 2024-07-12 204425](https://github.com/user-attachments/assets/d3adbafd-7377-49a7-9e6f-b3a9061a1f51)
![屏幕截图 2024-07-12 204353](https://github.com/user-attachments/assets/3aaff81e-5368-4a41-b274-6c7d4cfbd1de)

[sbi.json](https://github.com/user-attachments/files/16194335/sbi.json)

**Comment by 10Ring** [comment]
Hi @wuwuwuyue , thanks for your interest in our work. 

Our results are reported by training on the c0-raw version of FF++. We did not train on c23. 
In my opinion, working with c23 is a good try; however, besides try consulting our preprocessing steps, please also try to experiment some other things, e.g. hyper-params when you do SBI, new data augmentations, training (lr, etc). 
Many thanks.

**Comment by shui-tian-ju-shi** [comment]
Hello! After I start training, the accuracy on the validation set always remains at 50. As shown in the first picture. I found through debugging that in the second picture, the mask of the mixed image is 0. I don't know what went wrong. Can you give me some suggestions?
<img width="416" alt="image" src="https://github.com/user-attachments/assets/ee52da8b-80e0-4d92-9730-1daa418d4c13">
<img width="415" alt="image" src="https://github.com/user-attachments/assets/8c3b692e-b488-4dd1-9a8f-05cf4cf19ba2">

**Comment by userwxp** [comment]
Hi there ， i have been working on this for mouthes  ,can i see your code please!!!!!!! I appreciate this so much，i would love to communicate with you on issues involve preprocessing but if you don't have time for this ,can i see your code please ,can't find your email ,and this is mine 17782210150@163.com,if you see this ,please contact me ,thank you  so so much!!!!!

**Comment by wuwuwuyue** [comment]
> 你好！开始训练后，验证集的准确率始终保持在 50。如第一张图片所示。我通过调试发现，在第二张图中，混合图像的蒙版为 0。我不知道出了什么问题。您能给我一些建议吗？ <img alt="图像" width="416" src="https://private-user-images.githubusercontent.com/95734189/374765583-ee52da8b-80e0-4d92-9730-1daa418d4c13.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ2OTk2MjUsIm5iZiI6MTc0NDY5OTMyNSwicGF0aCI6Ii85NTczNDE4OS8zNzQ3NjU1ODMtZWU1MmRhOGItODBlMC00ZDkyLTk3MzAtMWRhYTQxOGQ0YzEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDE1VDA2NDIwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMxMjM1YzQzZTNmOWRiZjdkZTZiMzcwMzA1YWEyOGU4YzllN2Y3N2M1YWFlNzg3ZGVhMTRhNjQ2NjUwNjVmYzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GWsIMr3XDY6AEWUU24tMAsR5DxVoZDQyZcAE6AAqdXM"> <img alt="图像" width="415" src="https://private-user-images.githubusercontent.com/95734189/374765597-8c3b692e-b488-4dd1-9a8f-05cf4cf19ba2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ2OTk2MjUsIm5iZiI6MTc0NDY5OTMyNSwicGF0aCI6Ii85NTczNDE4OS8zNzQ3NjU1OTctOGMzYjY5MmUtYjQ4OC00ZGQxLTlhOGYtMDVjZjRjZjE5YmEyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDE1VDA2NDIwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlNDMxYzcxOGZiYWE1ODNjMmE2NGQyMDRjNjkzNWJmZjI2ZWFjMjVkZDA2ZTU4ZWY3NTYwYzMzNzU5MjE4N2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.w-dpQUbSD1mwBUMJXcdAvc8QREsc_qUOUxKd3FO9RgE">

**Comment by wuwuwuyue** [comment]
> 你好！开始训练后，验证集的准确率始终保持在 50。如第一张图片所示。我通过调试发现，在第二张图中，混合图像的蒙版为 0。我不知道出了什么问题。您能给我一些建议吗？ <img alt="图像" width="416" src="https://private-user-images.githubusercontent.com/95734189/374765583-ee52da8b-80e0-4d92-9730-1daa418d4c13.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ2OTk4NzMsIm5iZiI6MTc0NDY5OTU3MywicGF0aCI6Ii85NTczNDE4OS8zNzQ3NjU1ODMtZWU1MmRhOGItODBlMC00ZDkyLTk3MzAtMWRhYTQxOGQ0YzEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDE1VDA2NDYxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZjMzg0MWVlMjc2NzBlMmNhYzNmYzQ5ZTA5MWRhNTk4ZGU3ZGQwOWJmNTIwOGE2OGUyNTE1NTBhOTk5OTZmNTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.XUD8q1CxB0-RpiQKV6VirOS09QajnxfcFN7pbHCUM6g"> <img alt="图像" width="415" src="https://private-user-images.githubusercontent.com/95734189/374765597-8c3b692e-b488-4dd1-9a8f-05cf4cf19ba2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ2OTk4NzMsIm5iZiI6MTc0NDY5OTU3MywicGF0aCI6Ii85NTczNDE4OS8zNzQ3NjU1OTctOGMzYjY5MmUtYjQ4OC00ZGQxLTlhOGYtMDVjZjRjZjE5YmEyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDE1VDA2NDYxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYxZDg0NjY0Yjk1MDU2MTE3MTg5MGYyMDBiZGU0MGRkYTA1OGQ4NmU2M2NkMTM4YTg1YjY0MWFiNTgyNzNkMzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.U88oy0rTXtsSjh-T3BCG1LoRXyT3xlraWDrJXL6kZlE">

mask代码没什么问题，图像是h*w*3的，mask是h*w*1的，所以要取第一维度。你要是不放心，可以把训练图像和mask存下来看看有没有问题，不过记得用正确方式保存，这个代码还是没什么问题的，只要你按他的步骤来不会有错，不过训练时间比较久。
